src/SimpleDTO.php

```diff
@@ -350,6 +342,18 @@ protected function validatedData(): array
         return $result;
     }
 
+    /**
+     *  Check cast is callable or instance of Castable.
+     */
+    protected function getCast($cast, string $key, mixed $value): mixed
+    {
+        if($cast instanceof  \WendellAdriel\ValidatedDTO\Casting\Castable){
```

Comment:
No need to use the full namespace here

Reply:
https://github.com/WendellAdriel/laravel-validated-dto/pull/38/commits/b78e0601d705f6c1e65e1c20c155cef30cfd1a3e